mpyg321/mpyg321.py

```diff
@@ -252,6 +280,22 @@ def jump(self, pos):
         """Jump to position"""
         self.player.sendline("JUMP " + str(pos))
 
+    def volume(self, percent):
+        """Adjust player's volume"""
+        if self.player_version == "mpg123":
+            self.player.sendline("VOLUME {}".format(percent))
+
+    def silence(self):
+        """Silences the player"""
+        if self.player_version == "mpg123":
+            self.player.sendline("SILENCE")
+
+    def load_list(self, entry, filepath):
```

Comment:
Maybe we should add in the documentation of the function what the parameters are supposed to be?

Reply:
Yes, make sense!

- **entry** : integer - song number in the playlist
- **filepath** : string - full filename of a text file containg a list of songs, each line in the file is the full path of a mp3 file

Reply:
OK, I will add it in the code :)